DIY Peach!

```diff
@@ -0,0 +1,18 @@
+Guide for starting a Peach:
```

Comment:
As you mention in the PR, peach = working group, but please explain this somewhere in the document.